scripts/cc-cli/src/commands/send.ts

```diff
@@ -18,6 +20,7 @@ export function makeSendCommand() {
     "-f, --file [file-name]",
     "Specify file with mnemonic phrase to use for sending CTC"
   );
+  cmd.option("--scheme [scheme]", "Specify signature scheme to use", "sr25519");
```

Comment:
The way the if condition below is currently implemented is more akin to `--use-ecdsa` flag instead of specifying a scheme. 

I'm not aware what the possible scheme values are but sounds like a good idea to check if the value is valid.

Reply:
Agreed with Alex a `--use-ecdsa` flag is more aesthetic. But If we keep the scheme flag we should add to the description text all of the valid options:
`  cmd.option("--scheme [scheme]", "Specify signature scheme to use (sr25519, ecdsa)", "sr25519");`